src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -88,7 +108,7 @@
             <div class="vads-u-background-color--gray-light-alt vads-u-height--full medium-screen:vads-u-margin-x--1 medium-screen:vads-u-margin-y--0">
               <img alt="{{ promo.entity.fieldImage.entity.thumbnail.alt }}" src="{{ promo.entity.fieldImage.entity.thumbnail.url }}" />
               <h3 class="vads-u-padding-x--2">
-                <a href="{{ promo.entity.fieldPromoLink.entity.fieldLink.uri }}">
+                <a onclick="recordEvent({ 'event': 'clp-link-click', 'clp-section-category': 'What you can do', 'clp-section-title': '{{ fieldClpWhatYouCanDoHeader }}', 'clp-total-sections': '{{ clpTotalSections }}', 'clp-click-label': '{{ promo.entity.fieldPromoLink.entity.fieldLink.title }}' });" href="{{ promo.entity.fieldPromoLink.entity.fieldLink.uri }}">
```

Comment:
if there are potentially quotes or special characters here i think we need to use the `escape` filter like `'clp-section-title': '{{ fieldClpWhatYouCanDoHeader | escape }}'`

Reply:
Done! 🙂